std/path.d

```diff
@@ -3269,7 +3269,7 @@ body
                             /* Match for:
                              *   pattern[pi0..pi-1] ~ pattern[piRemain..$]
                              */
-                            if (pattmp.ptr == null)
+                            if (() @trusted { return pattmp.ptr; }() == null)
```

Comment:
I remember some argument a while back on what `array == null` should be. Don't remember the details.